### DISCUSSION TITLE
Creating .hic files

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3601] [Q&A] [answered]
Hello,
I couldn't find a way to generate .hic files in the documentation. Can someone please give me an example of how to generate .hic files and how to generate and SV inspector view.
Thanks

**Comment by cmdcolin** [comment, marked as answer]
I am not experienced in the generation pipeline, but .hic files are primarily produced by the juicebox toolkit https://github.com/aidenlab/Juicebox

For the SV inspector, a short tutorial is here https://jbrowse.org/jb2/docs/user_guides/sv_inspector_view/
